Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         .package(url: "https://github.com/yonaskolb/JSONUtilities.git", from: "4.2.0"),
         .package(url: "https://github.com/kylef/Spectre.git", from: "0.9.2"),
         .package(url: "https://github.com/onevcat/Rainbow.git", from: "3.0.0"),
-        .package(url: "https://github.com/tuist/XcodeProj.git", from: "8.12.0"),
+        .package(url: "https://github.com/tuist/XcodeProj.git", branch: "main"), // Will be supported in 8.13.0
```

Comment:
```suggestion
        .package(url: "https://github.com/tuist/XcodeProj.git", from: "8.13.0"),
```

Reply:
now released 🎉

Reply:
Yes, I changed the dependency to the release version 👍